### ISSUE TITLE
Issue with api pattern

### ISSUE THREAD

**Issue by hoangtaiki** [BiAtoms/Http.swift#16]
Server returns incorrect response when I define two GET request with nearly same pattern.
1. GET doctor_profile
API path: `/doctors/{id}/`
2. GET doctor_feedbacks
API path: `/doctors/{id}/feedbacks`

Problem: Call GET doctor_feedbacks but server return response for doctor_profile ( If you change adding order of two request server will works well)

Code to reproduce
```
let firstResponseString = "Doctor profile"
let secondResponseString = "Doctor feedback"

// Doctor profile
server.get("/doctors/{id}/") { request in
    return .ok(firstResponseString)
}

// Doctor feedbacks
server.get("/doctors/{id}/feedbacks") { request in
    return .ok(secondResponseString)
}

// Test for GET request
let ex1 = expectation(description: "test")
client.request("/doctors/{id}/feedbacks", method: .get)
    .responseString { r in
        XCTAssertEqual(r.value, secondResponseString)
        ex1.fulfill()
}
waitForExpectations()
```

**Comment by OrkhanAlikhanov** [comment]
If you swap the order it should work

**Comment by hoangtaiki** [comment]
Yes. I read your code and I understand the reason.

**Comment by OrkhanAlikhanov** [comment]
Released a fix at [v2.2.1](https://github.com/BiAtoms/Http.swift/releases/tag/2.2.1)

Thank you for reporting, your test case was helpful as well!
